### PR TITLE
Yarn 2, webpack 5.

### PR DESCRIPTION
**Pull request by internettrans**
In webpack 4, you specify webpack env vars via `--env.varName`. In webpack 5, it's `--env varName`. This change updates the language in the html file to be correct depending on which version of webpack you're using. I also added yarn 2 - from what I've heard this will still work for yarn 1 users after it's merged

### CODE REVIEW

File: lib/standalone-single-spa.js [hunk 1]
```diff
@@ -1,4 +1,12 @@
-const HtmlWebpackPlugin = require("safe-require")("html-webpack-plugin");
+const safeRequire = require("safe-require");
+const HtmlWebpackPlugin = safeRequire("html-webpack-plugin");
```
Comment (merceyz) [inline lib/standalone-single-spa.js]: Could you try removing safe-require and just using normal require with a try catch?

File: lib/standalone-single-spa.js [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
+let isWebpack5;
+
+try {
+  isWebpack5 = safeRequire("webpack").version.startsWith("5");
```
Comment (merceyz) [inline lib/standalone-single-spa.js]: Needs to be a peer dependency (and not using safeRequire)

**Review by merceyz — CHANGES_REQUESTED** [review]
Hey, I saw [the thread on twitter](https://mobile.twitter.com/Joelbdenning/status/1332440439496998913) and started working on PnP support for `single-spa` and was about to make a PR here but since you're already on it i'll leave my finding here. 

The use of `safe-require` breaks PnP support (and makes you rely on hoisting when using node_modules) because it runs the require on behalf of itself and not the caller and should be replaced with a simple require wrapped in `try catch`.

https://yarnpkg.com/advanced/rulebook#packages-should-only-ever-require-what-they-formally-list-in-their-dependencies

**Comment by internettrans** [comment]
Thanks for your review - i didn't know that about safeRequire. Will update.

**Comment by internettrans** [comment]
I've updated this with your feedback.

**Review by merceyz — APPROVED** [review]
Perfect, looks good!

**Pull request merged**